src/schoology-extractor/schoology_extractor/helpers/constants.py

```diff
@@ -21,4 +21,5 @@ class RESOURCE_NAMES:
     SUBMISSION = 'submissions'
     ENROLLMENT = "enrollment"
     ATTENDANCE_EVENTS = 'attendance_events'
-    USER_ACTIVITIES = 'user_activities'
+    DISCUSSION_REPLIES = 'discussion_replies'
+    DISCUSSIONS = 'discussions'
```

Comment:
Please run `black` to reformat. It will convert those apostrophes to quotation marks. Also run on the other files below.

Reply:
✔️